src/main/resources/plugin.yml

```
@@ -24,7 +24,7 @@ description: Customize Minecraft's mechanics with simple scripts written in plai
 
 authors: [Njol, Mirreski, 'SkriptLang Team', Contributors]
 
-website: https://skriptlang.github.io/Skript
+website: https://github.com/SkriptLang/Skript and https://docs.skriptlang.org
```

I know there's been discussion about making a proper home page, but I think using the docs for that purpose would be best for now, instead of linking both. I'd like others' opinions first though.

---

Agree, the docs currently have all the info needed, soon a homepage will be added.
```suggestion
website: https://docs.skriptlang.org
```

---

A better solution is using `https://www.skriptlang.org` which will be setup to redirect to the docs in the meantime until the home page is ready at the same endpoint

---

As of right now it redirects to the GitHub.